src/app/js/fields/wizard/StepUri.js

```diff
@@ -13,26 +12,16 @@ import StepUriAutogenerate from './StepUriAutogenerate';
 import StepValueColumn from './StepValueColumn';
 import StepUriConcat from './StepUriConcat';
 
-export const StepValueComponent = ({
-    datasetFields,
-    field,
-    handleTransformerChange,
-}) => (
+export const StepValueComponent = ({ handleTransformerChange }) => (
     <div>
-        <StepUriAutogenerate field={field} onChange={handleTransformerChange} />
-        <StepValueColumn
-            datasetFields={datasetFields}
-            field={field}
-            onChange={handleTransformerChange}
-        />
-        <StepUriConcat field={field} onChange={handleTransformerChange} />
+        <StepUriAutogenerate onChange={handleTransformerChange} />
```

Comment:
You remove a prop without changing components' propTypes, or is the propType field use elsewhere?

Reply:
it was unused props, unset in proptypes